### PR TITLE
[FE-268] Add missing columns to group table

### PR DESCRIPTION
**Pull request by AnCichocka**
https://oops2000.atlassian.net/browse/SCRUM-268

code is awful, but it works

### CODE REVIEW

**Comment by AnCichocka** [comment]
@Szyntos check if to fixed in points cell is ok or should I round it somehow different please

File: frontend/src/components/Group/table/GroupTable.tsx [hunk 1]
```diff
@@ -61,11 +137,9 @@ export const GroupTable = ({
...
                   {index + 1}. {row.student.fullName}
                 </TableCell>
-                {row.subcategories.map((subcategory) => (
-                  <TableCell
-                    key={`${row.student.id}-${subcategory.subcategoryId}`}
-                  >
-                    {subcategory.pure ?? EMPTY_FIELD_STRING}
+                {getRowValues(row).map((value, index) => (
+                  <TableCell key={`${index}`}>
+                    {value?.toFixed(2) ?? EMPTY_FIELD_STRING}
```
Comment (Szyntos) [inline frontend/src/components/Group/table/GroupTable.tsx:142]: if you don't truncate anything you shouldn't end up rounding those numbers -> they are rounded in the backend to 2 decimal places

**Review by Szyntos — APPROVED** [review]
yay <3

File: frontend/src/components/Group/table/GroupTable.tsx (hunk 1 above)
Comment (AnCichocka) [reply frontend/src/components/Group/table/GroupTable.tsx:142]: so I guess fixed is just fine, thanks

**Pull request merged**